onnxruntime/python/session.py

```diff
@@ -42,6 +42,7 @@ def _load_model(self, providers=[]):
 
         self._inputs_meta = self._sess.inputs_meta
         self._outputs_meta = self._sess.outputs_meta
+        self._inputs_meta_including_initializers = self._sess.inputs_meta_including_initializers
```

Comment:
make sure you set self._inputs_meta_including_initializers to None in _reset_session()